src/AccessTokenProvider.js

```diff
@@ -119,4 +135,10 @@ export default class AccessTokenProvider {
       .then(token => this.handleAccessToken(token, resolve))
       .catch(error => this.handleError(error, reject));
   }
+
+  isConfigValid() {
+    return this.config.isResponseUnauthorized &&
+      this.config.createAccessTokenRequest &&
+      this.config.parseAccessToken;
```

Comment:
I'm not using lodash, but I'll add it and use partial import with `import isFunction from 'lodash/isFunction';`